framework/db/schema/pgsql/CPgsqlSchema.php

```diff
@@ -166,7 +166,12 @@ protected function resolveTableNames($table,$name)
 	protected function findColumns($table)
 	{
 		$sql=<<<EOD
-SELECT a.attname, LOWER(format_type(a.atttypid, a.atttypmod)) AS type, d.adsrc, a.attnotnull, a.atthasdef,
+SELECT
+	a.attname,
+	LOWER(format_type(a.atttypid, a.atttypmod)) AS type,
+	pg_get_expr(adbin, adrelid) AS default_value_text,
```

Comment:
@cebe Do you still think the old name "adsrc" should be kept as an alias? It is only used inside of this function and in createColumn().

Reply:
yes, createColumn() could be extended.